# Represents a part/szene of a map with start- and endpoints of markings in szene

string szeneType
geometry_msgs/Point[] startPoints
geometry_msgs/Point[] endPoints
